roles/galera/templates/server.cnf.j2

```diff
@@ -95,6 +94,5 @@ wsrep_log_conflicts=ON
 [mariadb-10.0]
 
 [sst]
-encrypt = 2
```

Comment:
Not needed anymore?

Reply:
No, it complains with an warning that the property is deprecated.